zero/src/main/java/mutiny/zero/BackpressureStrategy.java

```diff
@@ -0,0 +1,10 @@
+package mutiny.zero;
+
+public enum BackpressureStrategy {
+    BUFFER,
```

Comment:
We may need to add javadoc. Actually we should here and in mutiny too.

Reply:
It's a draft 😉